src/scrapers/base-isracard-amex.ts

```diff
@@ -339,9 +339,9 @@ class IsracardAmexBaseScraper extends BaseScraperWithBrowser<ScraperSpecificCred
     this.page.on('request', (request) => {
       if (request.url().includes('detector-dom.min.js')) {
         debug('force abort for request do download detector-dom.min.js resource');
-        request.abort();
+        request.abort().then(() => {}).catch(() => {});
       } else {
-        request.continue();
+        request.continue().then(() => {}).catch(() => {});
```

Comment:
I think here we should ignore the ESLint error because this async call is inside a callback.

Even if it will fail, I expect the proccess to fail and not to ignore it by `.catch(() => {})`

Reply:
added void operator, which disabled the eslint rule